app/components/ad-slot.js

```diff
@@ -64,10 +57,6 @@ export default Component.extend(
       const ads = this.get('ads.module');
       const name = this.name;
 
-      if (this.noAds) {
-        return;
-      }
-
       if (this.insertOnViewportEnter) {
         ads.pushSlotToQueue(name);
```

Comment:
Hihi, this part doesn't make sense, right?
https://github.com/Wikia/mobile-wiki/pull/1821/files#diff-98564636413dff58d77852057b876c48R76-R77

Reply:
That's right, I removed `ads` service in favour of `Ads` module.